### PR TITLE
Improve Accessibility of Modals, update DaisyUI

### PR DESCRIPTION
**Pull request by Sharqiewicz**
### What:

Modals in the application currently do not follow accessibility best practices, which can lead to a confusing user experience, especially for users relying on keyboard navigation or assistive technologies.

### How:

✅ 💅  Update `DaisyUI` to `4.11.1` from `2.52.0`
✅ 💅  Update `TailwindCSS`
✅ 💅  Update `react-daisyui` to `5.0.0` from `3.1.2`
✅ 💅  Refactor components to use newest DaisyUI components and classes
✅ ⚠️ Override unwanted styles from DaisyUI that spoiled the look of the site (`src/index.css`)
✅ 🐛  Fix Buttons in the app to be disabled why loading (i.e. When an extrinsic is submitted - it was possible to submit it again and again - now it's not possible)
✅ 🐛  Fix AssetSelector `alt` property to show currently chosen asset
✅ 🦸  Refactor Dialog component
✅ 🦸  Extend Dashboard Assets page (in the Portfolio table) from 5 to 8

Closes: #451

### CODE REVIEW

**Comment by ebma** [comment]
## `yarn.lock` changes
### Summary
| Status | Count |
| :- | :-: |
| [<sub>ADDED</sub>](#) | 1 |
| [<sub>UPDATED</sub>](#) | 60 |
| [<sub>DOWNGRADED</sub>](#) | 1 |
| [<sub>REMOVED</sub>](#) | 3 |
<details>
<summary>Click to toggle table visibility</summary>
<br/>

| Name | Status | Previous | Current |
| :- | :-: | :-: | :-: |
| `@graphql-codegen/client-preset` | [<sub>UPDATED</sub>](#) | 4.2.5 | 4.2.6 |
| `@graphql-codegen/gql-tag-operations` | [<sub>UPDATED</sub>](#) | 4.0.6 | 4.0.7 |
| `@graphql-codegen/plugin-helpers` | [<sub>UPDATED</sub>](#) | 5.0.3 | 5.0.4 |
| `@graphql-codegen/typed-document-node` | [<sub>UPDATED</sub>](#) | 5.0.6 | 5.0.7 |
| `@graphql-codegen/typescript` | [<sub>UPDATED</sub>](#) | 4.0.6 | 4.0.7 |
| `@graphql-codegen/typescript-operations` | [<sub>UPDATED</sub>](#) | 4.2.0 | 4.2.1 |
| `@graphql-codegen/visitor-plugin-common` | [<sub>UPDATED</sub>](#) | 5.1.0 | 5.2.0 |
| `@graphql-tools/code-file-loader` | [<sub>UPDATED</sub>](#) | 8.1.1 | 8.1.2 |
| `@graphql-tools/git-loader` | [<sub>UPDATED</sub>](#) | 8.0.5 | 8.0.6 |
| `@graphql-tools/graphql-tag-pluck` | [<sub>UPDATED</sub>](#) | 8.3.0 | 8.3.1 |
| `@npmcli/arborist` | [<sub>UPDATED</sub>](#) | 7.5.1 | 7.5.2 |
| `@npmcli/config` | [<sub>UPDATED</sub>](#) | 8.3.1 | 8.3.2 |
| `@remix-run/router` | [<sub>UPDATED</sub>](#) | 1.16.0 | 1.16.1 |
| `@sigstore/bundle` | [<sub>UPDATED</sub>](#) | 2.3.1 | 2.3.2 |
| `@sigstore/protobuf-specs` | [<sub>UPDATED</sub>](#) | 0.3.1 | 0.3.2 |
| `@sigstore/sign` | [<sub>UPDATED</sub>](#) | 2.3.1 | 2.3.2 |
| `@sigstore/tuf` | [<sub>UPDATED</sub>](#) | 2.3.3 | 2.3.4 |
| `@sigstore/verify` | [<sub>UPDATED</sub>](#) | 1.2.0 | 1.2.1 |
| `@substrate/ss58-registry` | [<sub>UPDATED</sub>](#) | 1.47.0 | 1.48.0 |
| `@tanstack/react-table` | [<sub>UPDATED</sub>](#) | 8.16.0 | 8.17.3 |
| `@tanstack/table-core` | [<sub>UPDATED</sub>](#) | 8.16.0 | 8.17.3 |
| `@types/babel__traverse` | [<sub>UPDATED</sub>](#) | 7.20.5 | 7.20.6 |
| `@types/lodash` | [<sub>UPDATED</sub>](#) | 4.17.1 | 4.17.4 |
| `@types/react` | [<sub>UPDATED</sub>](#) | 18.3.1 | 18.3.2 |
| `braces` | [<sub>UPDATED</sub>](#) | 3.0.2 | 3.0.3 |
| `caniuse-lite` | [<sub>UPDATED</sub>](#) | 1.0.30001617 | 1.0.30001621 |
| `cli-table3` | [<sub>UPDATED</sub>](#) | 0.6.4 | 0.6.5 |
| `color` | [<sub>REMOVED</sub>](#) | 4.2.3 | - |
| `color-string` | [<sub>REMOVED</sub>](#) | 1.9.1 | - |
| `core-js-compat` | [<sub>UPDATED</sub>](#) | 3.37.0 | 3.37.1 |
| `culori` | [<sub>ADDED</sub>](#) | - | 3.3.0 |
| `daisyui` | [<sub>UPDATED</sub>](#) | 2.52.0 | 4.11.1 |
| `electron-to-chromium` | [<sub>UPDATED</sub>](#) | 1.4.762 | 1.4.777 |
| `fill-range` | [<sub>UPDATED</sub>](#) | 7.0.1 | 7.1.1 |
| `glob` | [<sub>UPDATED</sub>](#) | 10.3.14 | 10.3.16 |
| `immutable` | [<sub>UPDATED</sub>](#) | 4.3.5 | 4.3.6 |
| `is-arrayish` | [<sub>DOWNGRADED</sub>](#) | 0.3.2 | 0.2.1 |
| `jackspeak` | [<sub>UPDATED</sub>](#) | 2.3.6 | 3.1.2 |
| `jose` | [<sub>UPDATED</sub>](#) | 5.2.4 | 5.3.0 |
| `libnpmaccess` | [<sub>UPDATED</sub>](#) | 8.0.5 | 8.0.6 |
| `libnpmdiff` | [<sub>UPDATED</sub>](#) | 6.1.1 | 6.1.2 |
| `libnpmexec` | [<sub>UPDATED</sub>](#) | 8.1.0 | 8.1.1 |
| `libnpmfund` | [<sub>UPDATED</sub>](#) | 5.0.9 | 5.0.10 |
| `libnpmhook` | [<sub>UPDATED</sub>](#) | 10.0.4 | 10.0.5 |
| `libnpmorg` | [<sub>UPDATED</sub>](#) | 6.0.5 | 6.0.6 |
| `libnpmpack` | [<sub>UPDATED</sub>](#) | 7.0.1 | 7.0.2 |
| `libnpmpublish` | [<sub>UPDATED</sub>](#) | 9.0.7 | 9.0.8 |
| `libnpmsearch` | [<sub>UPDATED</sub>](#) | 7.0.4 | 7.0.5 |
| `libnpmteam` | [<sub>UPDATED</sub>](#) | 6.0.4 | 6.0.5 |
| `libnpmversion` | [<sub>UPDATED</sub>](#) | 6.0.1 | 6.0.2 |
| `micromatch` | [<sub>UPDATED</sub>](#) | 4.0.5 | 4.0.7 |
| `npm` | [<sub>UPDATED</sub>](#) | 10.7.0 | 10.8.0 |
| `npm-profile` | [<sub>UPDATED</sub>](#) | 9.0.2 | 10.0.0 |
| `nwsapi` | [<sub>UPDATED</sub>](#) | 2.2.9 | 2.2.10 |
| `path-scurry` | [<sub>UPDATED</sub>](#) | 1.11.0 | 1.11.1 |
| `picocolors` | [<sub>UPDATED</sub>](#) | 1.0.0 | 1.0.1 |
| `preact` | [<sub>UPDATED</sub>](#) | 10.21.0 | 10.22.0 |
| `react-daisyui` | [<sub>UPDATED</sub>](#) | 3.1.2 | 5.0.0 |
| `react-hook-form` | [<sub>UPDATED</sub>](#) | 7.51.4 | 7.51.5 |
| `react-router` | [<sub>UPDATED</sub>](#) | 6.23.0 | 6.23.1 |
| `react-router-dom` | [<sub>UPDATED</sub>](#) | 6.23.0 | 6.23.1 |
| `sass` | [<sub>UPDATED</sub>](#) | 1.77.0 | 1.77.2 |
| `sigstore` | [<sub>UPDATED</sub>](#) | 2.3.0 | 2.3.1 |
| `simple-swizzle` | [<sub>REMOVED</sub>](#) | 0.2.2 | - |
| `update-browserslist-db` | [<sub>UPDATED</sub>](#) | 1.0.15 | 1.0.16 |

</details>

File: src/index.css [hunk 1]
```diff
@@ -316,6 +321,52 @@ w3m-modal {
   white-space: nowrap;
 }
 
+/*  These classes are flawed in the version 4.11.1 of DaisyUI, so they are overwritten */
```
Comment (ebma) [inline src/index.css:324]: 'Flawed' in which sense? Just behaving differently than before or are they buggy?

**Review by ebma — APPROVED** [review]
Very nice refactoring, great job @Sharqiewicz 👍 

I also checked the dialogs and can confirm they improved. I noticed that we now have a nice animation for opening them. For closing not so much, but I think that's okay. (Maybe fadeout would be nice? not important though)

@prayagd  please also have another look at the dialogs.

File: src/index.css (hunk 1 above)
Comment (Sharqiewicz) [reply src/index.css:324]: @ebma  They completely change previous designs, causing style bugs throughout the app

Comment (ebma) [reply src/index.css:324]: Hmm okay, weird 🤔 but thanks for fixing it

**Comment by ebma** [comment]
## `yarn.lock` changes
### Summary
| Status | Count |
| :- | :-: |
| [<sub>ADDED</sub>](#) | 1 |
| [<sub>UPDATED</sub>](#) | 60 |
| [<sub>DOWNGRADED</sub>](#) | 1 |
| [<sub>REMOVED</sub>](#) | 3 |
<details>
<summary>Click to toggle table visibility</summary>
<br/>

| Name | Status | Previous | Current |
| :- | :-: | :-: | :-: |
| `@graphql-codegen/client-preset` | [<sub>UPDATED</sub>](#) | 4.2.5 | 4.2.6 |
| `@graphql-codegen/gql-tag-operations` | [<sub>UPDATED</sub>](#) | 4.0.6 | 4.0.7 |
| `@graphql-codegen/plugin-helpers` | [<sub>UPDATED</sub>](#) | 5.0.3 | 5.0.4 |
| `@graphql-codegen/typed-document-node` | [<sub>UPDATED</sub>](#) | 5.0.6 | 5.0.7 |
| `@graphql-codegen/typescript` | [<sub>UPDATED</sub>](#) | 4.0.6 | 4.0.7 |
| `@graphql-codegen/typescript-operations` | [<sub>UPDATED</sub>](#) | 4.2.0 | 4.2.1 |
| `@graphql-codegen/visitor-plugin-common` | [<sub>UPDATED</sub>](#) | 5.1.0 | 5.2.0 |
| `@graphql-tools/code-file-loader` | [<sub>UPDATED</sub>](#) | 8.1.1 | 8.1.2 |
| `@graphql-tools/git-loader` | [<sub>UPDATED</sub>](#) | 8.0.5 | 8.0.6 |
| `@graphql-tools/graphql-tag-pluck` | [<sub>UPDATED</sub>](#) | 8.3.0 | 8.3.1 |
| `@npmcli/arborist` | [<sub>UPDATED</sub>](#) | 7.5.1 | 7.5.2 |
| `@npmcli/config` | [<sub>UPDATED</sub>](#) | 8.3.1 | 8.3.2 |
| `@remix-run/router` | [<sub>UPDATED</sub>](#) | 1.16.0 | 1.16.1 |
| `@sigstore/bundle` | [<sub>UPDATED</sub>](#) | 2.3.1 | 2.3.2 |
| `@sigstore/protobuf-specs` | [<sub>UPDATED</sub>](#) | 0.3.1 | 0.3.2 |
| `@sigstore/sign` | [<sub>UPDATED</sub>](#) | 2.3.1 | 2.3.2 |
| `@sigstore/tuf` | [<sub>UPDATED</sub>](#) | 2.3.3 | 2.3.4 |
| `@sigstore/verify` | [<sub>UPDATED</sub>](#) | 1.2.0 | 1.2.1 |
| `@substrate/ss58-registry` | [<sub>UPDATED</sub>](#) | 1.47.0 | 1.48.0 |
| `@tanstack/react-table` | [<sub>UPDATED</sub>](#) | 8.16.0 | 8.17.3 |
| `@tanstack/table-core` | [<sub>UPDATED</sub>](#) | 8.16.0 | 8.17.3 |
| `@types/babel__traverse` | [<sub>UPDATED</sub>](#) | 7.20.5 | 7.20.6 |
| `@types/lodash` | [<sub>UPDATED</sub>](#) | 4.17.1 | 4.17.4 |
| `@types/react` | [<sub>UPDATED</sub>](#) | 18.3.1 | 18.3.2 |
| `braces` | [<sub>UPDATED</sub>](#) | 3.0.2 | 3.0.3 |
| `caniuse-lite` | [<sub>UPDATED</sub>](#) | 1.0.30001617 | 1.0.30001621 |
| `cli-table3` | [<sub>UPDATED</sub>](#) | 0.6.4 | 0.6.5 |
| `color` | [<sub>REMOVED</sub>](#) | 4.2.3 | - |
| `color-string` | [<sub>REMOVED</sub>](#) | 1.9.1 | - |
| `core-js-compat` | [<sub>UPDATED</sub>](#) | 3.37.0 | 3.37.1 |
| `culori` | [<sub>ADDED</sub>](#) | - | 3.3.0 |
| `daisyui` | [<sub>UPDATED</sub>](#) | 2.52.0 | 4.11.1 |
| `electron-to-chromium` | [<sub>UPDATED</sub>](#) | 1.4.762 | 1.4.777 |
| `fill-range` | [<sub>UPDATED</sub>](#) | 7.0.1 | 7.1.1 |
| `glob` | [<sub>UPDATED</sub>](#) | 10.3.14 | 10.3.16 |
| `immutable` | [<sub>UPDATED</sub>](#) | 4.3.5 | 4.3.6 |
| `is-arrayish` | [<sub>DOWNGRADED</sub>](#) | 0.3.2 | 0.2.1 |
| `jackspeak` | [<sub>UPDATED</sub>](#) | 2.3.6 | 3.1.2 |
| `jose` | [<sub>UPDATED</sub>](#) | 5.2.4 | 5.3.0 |
| `libnpmaccess` | [<sub>UPDATED</sub>](#) | 8.0.5 | 8.0.6 |
| `libnpmdiff` | [<sub>UPDATED</sub>](#) | 6.1.1 | 6.1.2 |
| `libnpmexec` | [<sub>UPDATED</sub>](#) | 8.1.0 | 8.1.1 |
| `libnpmfund` | [<sub>UPDATED</sub>](#) | 5.0.9 | 5.0.10 |
| `libnpmhook` | [<sub>UPDATED</sub>](#) | 10.0.4 | 10.0.5 |
| `libnpmorg` | [<sub>UPDATED</sub>](#) | 6.0.5 | 6.0.6 |
| `libnpmpack` | [<sub>UPDATED</sub>](#) | 7.0.1 | 7.0.2 |
| `libnpmpublish` | [<sub>UPDATED</sub>](#) | 9.0.7 | 9.0.8 |
| `libnpmsearch` | [<sub>UPDATED</sub>](#) | 7.0.4 | 7.0.5 |
| `libnpmteam` | [<sub>UPDATED</sub>](#) | 6.0.4 | 6.0.5 |
| `libnpmversion` | [<sub>UPDATED</sub>](#) | 6.0.1 | 6.0.2 |
| `micromatch` | [<sub>UPDATED</sub>](#) | 4.0.5 | 4.0.7 |
| `npm` | [<sub>UPDATED</sub>](#) | 10.7.0 | 10.8.0 |
| `npm-profile` | [<sub>UPDATED</sub>](#) | 9.0.2 | 10.0.0 |
| `nwsapi` | [<sub>UPDATED</sub>](#) | 2.2.9 | 2.2.10 |
| `path-scurry` | [<sub>UPDATED</sub>](#) | 1.11.0 | 1.11.1 |
| `picocolors` | [<sub>UPDATED</sub>](#) | 1.0.0 | 1.0.1 |
| `preact` | [<sub>UPDATED</sub>](#) | 10.21.0 | 10.22.0 |
| `react-daisyui` | [<sub>UPDATED</sub>](#) | 3.1.2 | 5.0.0 |
| `react-hook-form` | [<sub>UPDATED</sub>](#) | 7.51.4 | 7.51.5 |
| `react-router` | [<sub>UPDATED</sub>](#) | 6.23.0 | 6.23.1 |
| `react-router-dom` | [<sub>UPDATED</sub>](#) | 6.23.0 | 6.23.1 |
| `sass` | [<sub>UPDATED</sub>](#) | 1.77.0 | 1.77.2 |
| `sigstore` | [<sub>UPDATED</sub>](#) | 2.3.0 | 2.3.1 |
| `simple-swizzle` | [<sub>REMOVED</sub>](#) | 0.2.2 | - |
| `update-browserslist-db` | [<sub>UPDATED</sub>](#) | 1.0.15 | 1.0.16 |

</details>

**Review by prayagd — APPROVED** [review]
* Checked the dashboard length - LGTM
* Checked all dialogs from staking feature - stake, unstake, claim and from the transcations - LGTM
* Also checked the asset selection dropdown, asset shows up instead of random text - LGTM
* Tried the resubmitting option on button, now the buttons are disabled - LGTM
Great work @Sharqiewicz

**Pull request merged**